### DISCUSSION TITLE
CANNOT GET Error when trying to run Perplexica

### DISCUSSION THREAD

**Discussion by tristam15** [ItzCrazyKns/Vane#288] [Q&A] [answered]
Thank you for creating Perplexica. 

I'm getting this error when I try to run Perplexica. 
<img width="103" alt="image" src="https://github.com/user-attachments/assets/daeca136-693a-461f-9775-44b3f28e4751">

Can you tell me what is wrong here?

**Comment by ItzCrazyKns** [comment, marked as answer]
You are accessing the backend instead of the frontend, please check the port which you sued for the frontend and open it in the browser
